package.json

```diff
@@ -78,7 +78,8 @@
     "test-ci": "npm run check-licence && npm run lint -s && npm run cover",
     "test-repeat": "NODE_DEBUG=autobahn node test/index.js | FORCE_COLOR=1 tap-spec; while [ $? -eq 0 ]; do NODE_DEBUG=autobahn node test/index.js | FORCE_COLOR=1 tap-spec; done;",
     "test-cover": "istanbul cover test/index.js",
-    "view-cover": "opn ./coverage/index.html"
+    "view-cover": "opn ./coverage/index.html",
+    "prepublish": "git ls-files | grep '.js$' | grep -v 'bin/' | grep -v test | grep -v replacer.js | xargs node replacer.js"
```

Comment:
We need the same prepublish in tchannel :)

Which means tchannel is coupled to larch... which means... let's just figure it out.